src/brevitas_examples/llm/llm_quant/bias_corr.py

```diff
@@ -4,12 +4,13 @@
 """
 
 import torch
+from tqdm import tqdm
 
 from brevitas.graph.calibrate import bias_correction_mode
 
 
 @torch.no_grad()
 def apply_bias_correction(model, dataloader):
     with bias_correction_mode(model):
-        for inps in dataloader:
+        for inps in tqdm(dataloader):
```

Comment:
I think we need to add tqdm as required dependency (in examples requirements but at this point maybe everywhere)